apps/web/modules/ee/whitelabel/remove-branding/lib/project.ts

```
@@ -0,0 +1,56 @@
+} from "@/modules/ee/whitelabel/remove-branding/types/project";
+import { z } from "zod";
+import { prisma } from "@formbricks/database";
+import { projectCache } from "@formbricks/lib/project/cache";
+import { validateInputs } from "@formbricks/lib/utils/validate";
+import { ZId } from "@formbricks/types/common";
+import { ValidationError } from "@formbricks/types/errors";
+
+export const updateProjectBranding = async (
+  projectId: string,
+  inputProject: TProjectUpdateBrandingInput
+): Promise<Boolean> => {
```

Biome (1.9.4)

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)